src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts

```diff
@@ -0,0 +1,44 @@
+import { BORDER_RADIUS, COLORS, SPACINGS } from 'shared/style/constants'
+import styled from 'styled-components'
+
+export const StyledCardConfigContent = styled('div')`
+  background-color: ${COLORS.black};
+  padding: ${SPACINGS[3]};
+  border-radius: ${BORDER_RADIUS[1]};
+
+  .cardConfigContentHeadersContainer {
+    width: 100%;
+  }
+
+  .headersListContainer {
+    flex-wrap: wrap;
+    flex-direction: column;
+    max-width: 100%;
+    overflow-x: auto;
+  }
+
+  tr {
+    border-bottom: 1px solid #6a6a6a;
```

Comment:
Maybe you can add `#6a6a6a` in colors constant file

Reply:
Done